src/site/_site/Documentation/styles.scss

```diff
@@ -15,3 +15,9 @@
 .Documentation__section + .Documentation__section {
   margin-top: 1rem;
 }
+
+.Documentation__section h2 {
+  margin-top: 2rem;
```

Comment:
These styles are still being added unless otherwise overwritten. So in the case of the title component an h2 would get those margins b/c we don't overwrite them.
I would suggest either writing something like .documentation_section h2:not([class]) since all PF titles would require a class. The alternative could be leaving out styles entirely for the selector and require a class on them. It seems all the other heading selectors are using classes like .component-heading or .preview-title.

Reply:
@matthewcarleton  - I would like to change this a different way - we should be able to target specific sections of the documentation without needing to worry about PatternFly examples.  I added a docsection class which is only the markdown section with the H2 formatted correctly.

Reply:
ok that works for me. I feel like the workspace styles do need further consideration, I'm seeing inconsistencies/issues, but it makes more sense to wait until the dust settles on patternfly.org so we can better align the core/react workspace with the site. If that's the case we should only prioritize workspace changes that are affecting PF directly.

Reply:
I'd say this is ok to merge as long as we have further work down the road to address the styles overall.